tests/test_describe.py

```diff
@@ -101,24 +101,28 @@ def _predict(self, item):
     library = ModelLibrary(
         models=[SomeSimpleValidatedModelA, SomeComplexValidatedModelA]
     )
-    console = Console()
+    console = Console(no_color=True, force_terminal=False, width=130)
 
     with console.capture() as capture:
         library.describe(console=console)
 
-    if platform.system() != "Windows":
+    if platform.system() == "Windows" or platform.python_version().split(".")[:2] != [
```

Comment:
this check will silently ignore future python version.
can we improve it with a "greater than" comparison instead ?

Reply:
There is no guarantee that this test will function properly with Python 3.12 or later if there are further changes to Python's typing. I will make a commit to address this, and if the test fails on future Python versions, it may necessitate refactoring!